app/assets/stylesheets/application.css

```diff
@@ -14,6 +14,404 @@
  *= require_self
  */
 
-a:hover {
-  text-decoration: underline;
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#flash_alert,
+#flash_notice {
+  background-color: #3778c2 !important;
+  color: white !important;
+  font-weight: bold;
+  padding: 13px !important;
+  border-radius: 5px !important;
+}
+
+/* Base styles */
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+/* Container styles */
+.welcome {
+  background-color: #3778c2;
+  display: flex;
```

Comment:
Good job working on the project requirements, but it would be nice to stick to project requirements and design requirements you were required to design this application exactly as the given [Template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=&). Also you need to use the right font family for this project which is:

Correct font family:

For Headings Use : Proxima Nova Bold
For General Text: Proxima nova Light 

Like it says I the project requirements kindly follow this rule these must include colours as well.
Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

Colors
Typography: font face, size and weight.
Layout: composition and space between elements.

It would be nice to pay extra attention to detail.

Here is your design:

This is your Splash Screen:


![Student Splash screen](https://github.com/unorjikingsley/Budget-App/assets/79506105/d6b8d134-e2e1-4e5c-8485-0565a1c1b5d9)

This is your sign Up page:

![Student sign up page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/b962b17f-2991-4b81-bcc1-afc34ab6111c)

This is your login Page:


![Student login page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/bcb572e0-dfc3-4294-9467-f2dcfce7bfb6)

This is your Categories Page:
![Student Categories page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/9e6f4f2c-4cc6-4c68-84fb-62885477a4fa)

This is your Transaction Page:

![Transaction page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/1d36a157-6d2c-4b4d-9723-7a8a67c71c13)

This is your create new transaction Page:

![Student entity page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/864c257a-6a1c-4163-8d6e-cb52dd710e7e)
 You did an amazing job and I understand you put in a lot of time and effort in completing this project but it would be nice to follow design guidelines and rules.

Please design your work almost exactly like this design this design template:

This should include colours and fonts swell

Required design to follow:

Splash screen:

![Slash Screen](https://github.com/unorjikingsley/Budget-App/assets/79506105/fac4e844-53d2-4f45-96b9-1542eeb2f2fa)

Sign up:

![Sign up](https://github.com/unorjikingsley/Budget-App/assets/79506105/684e53e9-9833-46ff-88c7-cdc461a43c65)

Login Page:

![Login Page ](https://github.com/unorjikingsley/Budget-App/assets/79506105/17eda1da-1929-400b-ac61-056b53a6aa94)

Transaction Page:
![transacs](https://github.com/unorjikingsley/Budget-App/assets/79506105/4e3bca00-7e1b-4800-aa90-7ad2cef2d419)

![General design](https://github.com/unorjikingsley/Budget-App/assets/79506105/8902ae18-d434-4054-8cd3-507f47e0a35b)

Please pay more attention to detail.